app/views/items/_browse_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= t "browse.title", default: "Browse" %></h1>
+
+<p>
+  <%= link_to t("browse.all_link", default: "All Items in the Archive"), prefix_path(route_path),
```

Comment:
Should this text be "All Items in the collection"? Maybe this was copied from WCA? :smile: